xbmc/FileItem.h

```diff
@@ -583,6 +583,16 @@ class CFileItem :
   int m_iHasLock; // 0 - no lock 1 - lock, but unlocked 2 - locked
   int m_iBadPwdCount;
 
+  enum TITLES_JOB
+  {
+    TITLES_JOB_MAIN_TITLE = 0, // Default
+    TITLES_JOB_ALL_TITLES = 1,
+    TITLES_JOB_SINGLE_EPISODE = 2,
+    TITLES_JOB_ALL_EPISODES = 3
+  };
+
+  int m_titlesJob;
```

Comment:
Don’t know where this belongs, but this is definitely not a property of a file item. Maybe @notspiff has some thoughts?

Reply:
Thanks. I still need a flag to show if a FileItem refers to multiple episodes (for browsing through Video->Files) but I've simplified it in the lastest push.